completion_test.php

```diff
@@ -76,7 +76,7 @@ function tincanlaunch_get_completion_state_test($course,$cm,$userid,$type) {
     	$areAnyStatementsReturned = tincanlaunch_check_statements($tincanlaunchsettings['tincanlaunchlrsendpoint'], $tincanlaunchsettings['tincanlaunchlrslogin'], $tincanlaunchsettings['tincanlaunchlrspass'], $tincanlaunchsettings['tincanlaunchlrsversion'], $tincanlaunch->tincanactivityid, tincanlaunch_getactor(), $tincanlaunch->tincanverbid);
 		
 		echo ('<p>LRS response:</p><pre>');
-		print_r($areAnyStatementsReturned y);
+		print_r($areAnyStatementsReturned);
```

Comment:
This whole block (4 lines) looks like debugging code that's ben mistakenly left in. It can be deleted.